src/odb_loose.c

```diff
@@ -273,7 +273,7 @@ static int read_loose_packlike(git_rawobj *out, git_buf *obj)
 static int read_loose_standard(git_rawobj *out, git_buf *obj)
 {
 	git_zstream zstream = GIT_ZSTREAM_INIT;
-	unsigned char head[MAX_HEADER_LEN], *body = NULL;
+	unsigned char head[MAX_HEADER_LEN] = {0}, *body = NULL;
```

Comment:
Does zlib access these bytes before writing into the provided buffers? Or is it that zlib may write short, leaving the remainder uninitialized?

Reply:
Ah this was a workaround that I did before cherry-picking https://github.com/madler/zlib/pull/393, but it's not needed now that the root cause has been fixed. Removed.